doc/releases/release-notes-3.3.rst

```diff
@@ -0,0 +1,213 @@
+:orphan:
+
+.. _zephyr_3.3:
+
+Zephyr 3.3.0 (Working Draft)
+############################
+
```

Comment:
Would you mind adding the 

```
We are pleased to announce the release of Zephyr version 3.2.0.

Major enhancements with this release include:
```

section as shown in:
https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/main/doc/releases/release-notes-3.2.rst

Reply:
Otherwise people forget to add highlights. Thanks!

Reply:
Fixed